src/cli/command-workspace.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as chalk from 'chalk';
+import * as fs from 'fs';
+import * as path from 'path';
+import {Workspace} from 'polymer-workspaces';
+
+import {CliOptions} from '../cli';
+import {convertWorkspace} from '../convert-workspace';
+
+const githubTokenMessage = `
+You need to create a github token and place it in a file named 'github-token'.
+The token does not need any permissions.
+
+Generate a token here:   https://github.com/settings/tokens
+
+Then:
+
+echo 'PASTE TOKEN HEX HERE' > ./github-token
+`;
+
+
+/**
+ * Checks for github-token in the RunnerOptions and if not specified, will look
+ * in the github-token file in working folder.  If that doesn't exist either,
+ * we message to the user that we need a token and exit the process.
+ */
+function loadGitHubToken(): string|null {
```

Comment:
seem like this could move into polymer-workspaces

Reply:
I like that the library itself is agnostic to how the token is loaded. That makes it flexible enough for scripts, CLIs, etc.

It could be interesting as a helper function though, to at least make it easier to build CLIs.